### PR TITLE
Refreshment

### PR DESCRIPTION
**Pull request by fatadel**
In the first 2 commits, I have fixed typos and grammar mistakes in the narration as well as finished the last paragraph which wasn't complete.
In the other 2 commits, I have corrected some instructions, code snippets and also made images up to date so that they reflect the current TD specification.

### CODE REVIEW

**Comment by sebastiankb** [comment]
@egekorkan will review this PR

File: content/hands-on-intro-raspberry.md [hunk 1]
```diff
@@ -90,10 +89,11 @@ wot@node.wot:~$ npm version
 
 ### Cloning the repository 
 
-The reference implementation of the Web of Things is published in github as node.wot project at [github](https://github.com/eclipse/thingweb.node-wot) . The project is part of eclipse foundation.
+The reference implementation of the Web of Things is published in github as node.wot project at [github](https://github.com/eclipse/thingweb.node-wot) . The project is part of Eclipse Foundation.
```
Comment (egekorkan) [inline content/hands-on-intro-raspberry.md:92]: of the Eclipse ...

File: content/hands-on-intro-raspberry.md [hunk 2]
```diff
@@ -103,13 +103,14 @@ wot@node.wot:~$ git clone https://github.com/eclipse/thingweb.node-wot
 
 ### Installation 
 
-The installation process is composed from mainly two steps. The first step is installs the nodeJS dependences. The second step installs and build the node.wot project.
+The installation process is composed of mainly two steps. The first step installs the nodeJS dependences. The second step installs and builds the node.wot project.
 
 #### Installing the packages
 
-Node.wot requires dependencies such as Typescript, lerna, … . by running the command below, those dependencies are installed:
+Node.wot requires dependencies such as Typescript, lerna and etc. By running the command below, those dependencies are installed:
```
Comment (egekorkan) [inline content/hands-on-intro-raspberry.md:110]: Don't use and before etc.

File: content/hands-on-intro-raspberry.md [hunk 3]
```diff
@@ -118,7 +119,7 @@ wot@node.wot:~$ npm install
 
 #### Building the libraries 
 
-As the node.wot project is mainly developed Typescript, “tsc” needs to be executed to trans-compile that code to JavaScript. The command bellows builds the projects and its packages:
+As the node.wot project is mainly developed using Typescript, "tsc" needs to be executed to trans-compile that code to JavaScript. The command bellow builds the project and its packages:
```
Comment (egekorkan) [inline content/hands-on-intro-raspberry.md:122]: tsc command needs to be...

Comment (egekorkan) [reply content/hands-on-intro-raspberry.md:122]: The command below (not two l) builds the project source code under the packages

File: content/hands-on-intro-raspberry.md [hunk 4]
```diff
@@ -3,60 +3,59 @@
...
 
 + Uniform Resource Identifier (URI) for identifying devices and their descriptions
 + A wide variety of existing protocols for accessing the devices
-+ A description of the description as metadata used both for interoperability and presentability proposes
++ A description of the description as metadata used both for interoperability and presentability purposes
 
 ## WoT Architecture
 
-The central component of the WoT architecture is “WoT Servient” which is functional virtual device. A Servient extends the IoT physical devices by providing standard access and controls capabilities to the devices.
+The central component of the WoT architecture is "WoT Servient" which is functional virtual device. A Servient extends the IoT physical devices by providing standard access and control capabilities to the devices.
```
Comment (egekorkan) [inline content/hands-on-intro-raspberry.md:14]: A Thing does not have to be a Servient.

**Comment by fatadel** [comment]
@egekorkan 
Well, regarding most of the comments - I didn't write this article myself. I was just trying to fix typos and some basic grammar mistakes. I can definitely rewrite the whole tutorial the way I would do it, but I don't think that was the task.

**Comment by egekorkan** [comment]
Thank you for improving this tutorial, it is very valuable for the beginners :)

Could you make sure that:

- Node.js is written always like this
- Raspberry Pi Zero W is always written like this
- GitHub is always written like this
- It is Thing Description and not Thing**s** Description
- Every sentence is a new line in the md document so that upcoming changes to a single sentence does not highlight the whole paragraph. It makes it very difficult to review.
- Property, Action, Event are affordances. You use the word section
- CoAP is written like this
- Scripting API is  like this
- Servient like this (you have Serviant at some point)
- Thing always starts with a capital T
- node-wot is always like this and not node.wot

Additional feedback:
- This tutorial is not limited to RPi Zero W. Maybe you can just say Raspberry Pi in the text and in the beginning say that we prepared this tutorial with a Raspberry Pi
- Typescript and lerna are dependencies in the root level of the project but actually every package has its own dependencies.  Maybe you can say that npm install actually calls npm install in the children packages
- You say "W3C draft document available" but it is now a Recommendation.
- We do not add properties, actions, events anymore outside of produce() method. You use "Now that we have our property exposed, we can start adding our actions. An action as added to Serviant as described below:"
- I wouldn't give `https://github.com/w3c/wot` as a reference since it is just a data dump. Give either the main page or IG page for further references
- I find the title `Standard description` really weird. Should the reader understand that is where the standard is or where you would a a standard/basic TD
- Do not mix the servient that node-wot has and the definition servient from the Architecture spec.

**Comment by egekorkan** [comment]
> @egekorkan
> Well, regarding most of the comments - I didn't write this article myself. I was just trying to fix typos and some basic grammar mistakes. I can definitely rewrite the whole tutorial the way I would do it, but I don't think that was the task.

I think that with my comment above, you will see that there are more typos to be corrected. Otherwise, for things that are not typos, I leave the decision to @sebastiankb .

**Comment by sebastiankb** [comment]
I think there is no need to rewrite this tutorial. The intention was to update the content regarding the latest TD version and make sure that the commands are still valid with the latest node-wot version.

**Comment by sebastiankb** [comment]
A typo fix and some clarification of the spelling, as mentioned above in @egekorkan comment, would also be useful if we are refreshing the tutorial anyway.

**Comment by fatadel** [comment]
@sebastiankb I did the fixes and spelling clarifications without any rewriting, so it's now again ready for review (@egekorkan). From my side, there are some open questions left:

1. Do I need to create faster and more accurate gif videos?
2. Shall we change the part which talks about supported/recommended Node.js versions (imho outdated now)?
3. There is a description for the `wpa_supplicant.conf` file but no any info for `ssh`, although they both are said to create. Shall we add that as well?
4. More meaningful page title? Now it's just "thingweb - ".

I think that's it. I'll add more if I come up with any.

**Comment by sebastiankb** [comment]
Many thanks @fatadel. I thinks, your 4 points are valid and should discuss in a new PR. I'm going to merge this PR to have now new updated version so far.

**Pull request merged**